### PR TITLE
add elementUpdated to `testing` package exports

### PR DESCRIPTION
**Pull request by SampsonCrowley**

### CODE REVIEW

File: packages/testing-helpers/README.md [hunk 1]
```diff
@@ -105,6 +105,30 @@ Waits for `x` ms via `setTimeout`;
 await aTimeout(10); // would wait 10ms
 ```
 
+### elementUpdated
+
+If you want to testing attribute and property changes, and an easy way to wait for those changes to propagate, you can import the `elementUpdated` helper (also available directly in the `testing` package)
+
+```
```
Comment (daKmoR) [inline packages/testing-helpers/README.md]:
```suggestion
```js
```

File: packages/testing-helpers/README.md [hunk 2]
```diff
@@ -105,6 +105,30 @@ Waits for `x` ms via `setTimeout`;
...
+import { fixture, elementUpdated } from '@open-wc/testing-helpers';
+import '../my-component.js';
+
+describe('Attributes', () => {
+  describe('.title', () => {
+    ...
```
Comment (daKmoR) [inline packages/testing-helpers/README.md]:
```suggestion
    // ...
```
for code highlighting

File: packages/testing-helpers/README.md [hunk 3]
```diff
@@ -105,6 +105,30 @@ Waits for `x` ms via `setTimeout`;
...
+    it('is bound to the `title` attribute', async () => {
+      const el = await fixture('<my-component title="test"></my-component>');
+      expect(el.title).to.eq('test');
+
+      el.title = "test 2"
+      await elementUpdated(el)
+      expect(el).dom.to.equal(`<my-component title="test 2"></my-component>`);
+    });
+    ...
```
Comment (daKmoR) [inline packages/testing-helpers/README.md]:
```suggestion
    // ...
```

**Comment by daKmoR** [comment]
looks good - only small remark to improve code highlighting 👍 

the ci error for snyk you can ignore - it's not related to this change 🙈

**Comment by SampsonCrowley** [comment]
@daKmoR, just noticed a typo in the first paragraph, ("if you want to `testing`", instead of "if you want to `test`")

I'll add your suggestions and push again

**Comment by daKmoR** [comment]
thank you for this contribution 🤗 

ci will release the changes shortly 👍

**Pull request merged**